connectors/sink-feishu/README.md

```diff
@@ -116,14 +120,83 @@ tell Feishu Sink.
-to determine how to process event
+to determine how to process event.
+
+| Attribute         | Required | Examples               | Description                                                                                                                                                  |
+|:------------------|:--------:|------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| xvfeishuservice   | **YES**  | bot                    | which Feishu Service the event sent for                                                                                                                      |
+| xvfeishumsgtype   | **YES**  | text                   | which Message Type the event convert to                                                                                                                      |
+| xvfeishuchatgroup |    NO    | test_bot               | which Feishu chat-group the event sent for, the value should associate with you wrote in configuration, if `dynamic_route=false`, this attribute can't empty |
+| xveishuboturls    |    NO    | bot1,bot2,bot3         | dynamic webhook urls, use  `,` to separate multiple urls.                                                                                                    |
+| xvfeishubotsigns  |    NO    | signature1,,signature3 | dynamic webhook signatures, use  `,` to separate multiple signatures.                                                                                        |
+
+**the number of urls represented by `xveishuboturls` must equal to the number of signatures represented by `xveishuboturls`**
+
+### Chat Bot Dynamic Webhook
+
+In some cases, users can't make how many bots there have or wanner send one message to multiple groups, which means they need to dynamically send message to
+Feishu Bot Service, `Chat Bot Dynamic Webhook` helps users do that.
+
+in `config.yml`, set `dynamic_route=true` to enable this feature, otherwise `xveishuboturls` and `xvfeishubotsigns` will be ignored.
+
+```yaml
+enable: ["bot"]
+bot:
+  dynamic_route: true
+    webhooks:
+    - chat_group: "bot_predefined"
+      url: "https://open.feishu.cn/open-apis/bot/v2/hook/xxxxx"
+      signature: "xxxxxx"
+```
+
+```shell
+curl --location --request POST 'localhost:31080' \
+--header 'Content-Type: application/cloudevents+json' \
+--data-raw '{
+    "id": "53d1c340-551a-11ed-96c7-8b504d95037c",
+    "source": "sink-feishu-quickstart",
+    "specversion": "1.0",
+    "type": "quickstart",
+    "datacontenttype": "application/json",
+    "time": "2022-10-26T10:38:29.345Z",
+    "xvfeishuservice": "bot",
+    "xvfeishumsgtype": "text",
+    "xveishuboturls": "https://open.feishu.cn/open-apis/bot/v2/hook/xxx,https://open.feishu.cn/open-apis/bot/v2/hook/xxx,https://open.feishu.cn/open-apis/bot/v2/hook/xxx",
```

Comment:
does the pattern all prefix "https://open.feishu.cn/open-apis/bot/v2/hook", if true, it can be put in the config, so the user only needs to set xxx to reduce URL